### ISSUE TITLE
Incomplete documentation

### ISSUE THREAD

**Issue by stAn47** [sendinblue/APIv3-php-library#146]
Hello, i am trying create a Joomla plugin out of your examples listed here:
https://github.com/sendinblue/APIv3-php-library/blob/master/docs/Api/ContactsApi.md#addcontacttolist

but they seem to be very incomplete - can you pls adjust them to use some examples datas to include some real world inputs such as: 
- name: first name + middle name + lastname (possibly others that could be used in templates)
- email address

even though your company seems to provide email services, the examples don't include a single place where the email address gets registered to your site: 

```
$contactEmails = new \SendinBlue\Client\Model\RemoveContactFromList(); // \SendinBlue\Client\Model\RemoveContactFromList | Emails addresses OR IDs of the contacts
```

does the comment means to enter the array of emails within the constructor? or anything else ? same for all other API fucntions using this. 

in short what i really need is per ecommerce usage: 
- see if email address is already registred (at all) or blacklisted  (checkbox for newsletter is on or off by result of this)
- create/update email address against a list  (assign user to specific lists per various logic at backend)
- remove email address from all lists and blacklist it   (if checkbox was unticked make sure to remove the user from all lists)

thank you, best regards, stan

**Comment by rajatsib** [comment]
Hi @stAn47 

The model details are mentioned[ here](https://github.com/sendinblue/APIv3-php-library/blob/master/docs/Model/RemoveContactFromList.md).

Below is an example that you can refer

<?php
require_once(__DIR__ . '/vendor/autoload.php');

$config = SendinBlue\Client\Configuration::getDefaultConfiguration()->setApiKey('api-key', 'YOUR API KEY');

$apiInstance = new SendinBlue\Client\Api\ContactsApi(
    new GuzzleHttp\Client(),
    $config
);
$listId = 1;
$contactIdentifiers = new \SendinBlue\Client\Model\RemoveContactFromList();
$contactIdentifiers['emails'] = array('example@example.com');

try {
    $result = $apiInstance->removeContactFromList($listId, $contactIdentifiers);
    print_r($result);
} catch (Exception $e) {
    echo 'Exception when calling ContactsApi->removeContactFromList: ', $e->getMessage(), PHP_EOL;
}
?>
